Common/Utilities.cs

```diff
@@ -24,5 +25,13 @@ public static void LogAssemblyVersion(Assembly assembly, string description, ILo
 
             logger.LogInfo("{0} {1}", description, assembly.GetName().Version);
         }
+
+        public static string CreateTempDirectory(string dirName)
+        {
+            string newPath = Path.GetTempPath();
+            newPath = Path.Combine(newPath, ".sqsdk", dirName);
+            Directory.CreateDirectory(newPath);
+            return newPath;
+        }
```

Comment:
Miscellaneous: I've started tidying up where the temporary files are create so there are under single file ".sqssk" folder in temp directory. Previously different components were creating temp directories in a variety of locations. There may still be a few temp directories that need to be fixed up.